composables/useNft.ts

```diff
@@ -52,7 +52,7 @@ async function getProcessMetadata(nft: NFTWithMetadata) {
   const getAttributes = () => {
     const hasMetadataAttributes =
       metadata.attributes && metadata.attributes.length > 0
-    const attr = nft?.meta?.attributes || []
+    const attr = nft?.attributes || nft?.meta?.attributes || []
```

Comment:
I would probably do a merge of properties?

Reply:
Done. Now they are merging to a properties list.